plugin/NERD_commenter.vim

```diff
@@ -216,7 +216,7 @@ let s:delimiterMap = {
     \ 'hog': { 'left': '#' },
     \ 'hostsaccess': { 'left': '#' },
     \ 'htmlcheetah': { 'left': '##' },
-    \ 'htmldjango': { 'left': '<!--', 'right': '-->', 'leftAlt': '{#', 'rightAlt': '#}' },
+    \ 'htmldjango': { 'left': '{% comment %}', 'right': '{% endcomment %}', 'leftAlt': '{% comment %}', 'rightAlt': '{% endcomment %}' },
```

Comment:
Is this an embedded language used inside of HTML? If so the alternate comment syntax should be for the language it co-exists with, perhaps something like this:

```suggestion
    \ 'htmldjango': { 'left': '{% comment %}', 'right': '{% endcomment %}', 'leftAlt': '<!--', 'rightAlt': '-->' },
```